pkg/kritis/util/whitelist_test.go

```diff
@@ -52,17 +52,27 @@ func Test_RemoveGloballyWhitelistedImages(t *testing.T) {
 	}
 }
 
-func Test_imageInWhitelist(t *testing.T) {
+func Test_ImageInWhitelist(t *testing.T) {
 	tests := []struct {
 		name     string
 		image    string
 		expected bool
 	}{
 		{
-			name:     "test image in whitelist",
+			name:     "test image with tag in whitelist",
 			image:    "gcr.io/kritis-project/kritis-server:tag",
 			expected: true,
 		},
+		{
+			name:     "test image with digest in whitelist",
+			image:    "gcr.io/kritis-project/kritis-server@sha256:0000000000000000000000000000000000000000000000000000000000000000",
+			expected: true,
+		},
+		{
+			name:     "test public image in whitelist",
+			image:    "nginx@sha256:0000000000000000000000000000000000000000000000000000000000000000",
+			expected: true,
```

Comment:
How about including image whitelist here?

Reply:
Fixed in b314ba5